src/Discovery/DiscoveryDetails/DiscoveryDetails.tsx

```diff
@@ -210,6 +211,10 @@ const tabField = (fieldConfig: TabFieldConfig, discoveryConfig: DiscoveryConfig,
     && resourceFieldValue[0]
     && resourceFieldValue[0].length !== 0
   ) {
+    if (fieldConfig.type === 'dataDownloadList') {
+      return <DataDownloadList sourceFieldData={resourceFieldValue} />;
```

Comment:
this component needs some vertical padding in order to be separated from other siblings, consider adding `fieldCls` to it

Reply:
Added vertical padding using a new class for the DataDownload list component. Wasn't able to add vertical padding using fieldCls without other side effects.